apps/src/templates/VisualizationOverlay.js

```diff
@@ -64,7 +64,7 @@ VisualizationOverlay.prototype.render = function (intoElement, nextProps) {
   var oldProps = $.extend({}, this.props_);
   $.extend(this.props_, nextProps);
 
-  if (this.props_.scale !== oldProps.scale) {
+  if (this.props_.scale !== oldProps.scale || !this.screenSpaceToAppSpaceTransform_) {
```

Comment:
I suspect it is.  I'm not sure why this has never come up in App Lab, but when I dropped the overlay into gamelab it failed to calculate this transform on first render.